PowerEditor/src/WinControls/Preference/preference_rc.h

```diff
@@ -480,4 +480,5 @@
 	#define IDC_CHECK_PERFORMANCE_ALLOWSMARTHILITE      (IDD_PREFERENCE_SUB_PERFORMANCE + 9)
 	#define IDC_CHECK_PERFORMANCE_DEACTIVATEWORDWRAP    (IDD_PREFERENCE_SUB_PERFORMANCE + 10)
 	#define IDC_CHECK_PERFORMANCE_ALLOWCLICKABLELINK    (IDD_PREFERENCE_SUB_PERFORMANCE + 11)
+	#define	IDC_CHECK_SUPPRESS2GBWARNING				(IDD_PREFERENCE_SUB_PERFORMANCE + 12)

```

Comment:
Rename `IDC_CHECK_SUPPRESS2GBWARNING` to `IDC_CHECK_PERFORMANCE_SUPPRESS2GBWARNING`

Reply:
Not LargeFileRestriction here? (curious)